.gitignore

```diff
@@ -62,3 +62,4 @@ windows/flutter/generated_plugin_registrant.cc
 windows/flutter/generated_plugin_registrant.h
 windows/flutter/generated_plugins.cmake
 pubspec.lock
+*.lock
```

Comment:
Why previous line is not enough?

Reply:
This line was added in your last commit, and as I didn't pull, I had a merge conflict. So I just added this line to my gitignore as well

Reply:
deleted *.lock from gitignore